recipes-debian/openjdk/openjdk-11-jre-headless_debian.bb

```diff
@@ -0,0 +1,176 @@
+    --disable-precompiled-headers \
+    --with-jvm-features=zgc,shenandoahgc \
+    --disable-ccache \
+    --with-debug-level=release \
+    --with-native-debug-symbols=zipped \
+    --disable-warnings-as-errors \
+    --disable-javac-server \
+    --with-stdc++lib=dynamic \
+    --with-giflib=system \
+    --with-libpng=system \
+    --with-zlib=system \
+    --with-libjpeg=system \
+    --with-lcms=system \
+    --with-pcsclite=system \
+    --with-harfbuzz=system \
+    --with-extra-cflags='${CFLAGS} ${EXTRA_CFLAGS}' \
+    --with-extra-cxxflags='${CXXFLAGS} ${EXTRA_CFLAGS}' \
+    --with-extra-ldflags='${LDFLAGS} ${EXTRA_LDFLAGS}' \
+    --with-sysroot=${STAGING_DIR_HOST} \
+    --enable-option-checking=yes \
+    --with-abi-profile=aarch64 \
+    --with-cpu-port=aarch64 \
+    --enable-headless-only=yes \
+    --enable-nss \
+"
+
+EXTRA_BUILD_ENV = "MAKE_VERBOSE=y QUIETLY= LOG=debug IGNORE_OLD_CONFIG=true LIBFFI_LIBS=-lffi_pic JOBS=${@oe.utils.cpu_count()}"
+
+do_configure () {
+    cd ${S}
+    rm -rf ${S}/build
+    mkdir ${S}/build
+    cd ${S}/build
+    oe_runconf
+}
+
+do_compile () {
+    cd ${B}
+    ${EXTRA_BUILD_ENV} oe_runmake images
+}
+
+PACKAGES = "${PN} ${PN}-dbg"
+
+do_install () {
+    mkdir -p ${D}${DEBIAN_JDK_BASE_DIR}
+    mkdir -p ${D}${DEBIAN_JDK_SYSCONF_DIR}
+    rm -rf ${D}${DEBIAN_JDK_BASE_DIR}/jmods
+    cp -rp ${B}/images/jdk/bin ${D}${DEBIAN_JDK_BASE_DIR}/bin
+    find ${D}${DEBIAN_JDK_BASE_DIR}/bin -type f -not -name 'java' -not -name 'jjs' \
+    -not -name 'keytool' -not -name 'pack200' -not -name 'rmid' -not -name 'rmiregistry' \
+    -not -name 'unpack200' | xargs -r rm -f
+    cp -rp ${B}/images/jdk/conf ${D}${DEBIAN_JDK_BASE_DIR}/conf
+    cp -rp ${B}/images/jdk/legal ${D}${DEBIAN_JDK_BASE_DIR}/legal
+    cp -rp ${B}/images/jdk/release ${D}${DEBIAN_JDK_BASE_DIR}
+    cp -rp ${B}/images/jdk/lib ${D}${DEBIAN_JDK_BASE_DIR}/lib
+    rm -f ${D}${DEBIAN_JDK_BASE_DIR}/lib/libjawt.so
+    rm -f ${D}${DEBIAN_JDK_BASE_DIR}/lib/src.zip
+    rm -f ${D}${DEBIAN_JDK_BASE_DIR}/conf/management/jmxremote.password.template
+    find ${D}${DEBIAN_JDK_BASE_DIR} -name '*.debuginfo' -o -name '*.diz' \
+     -o -name 'ADDITIONAL_LICENSE_INFO' -o -name 'LICENSE' | xargs -r rm -f
+    chown -R root:root ${D}${DEBIAN_JDK_BASE_DIR}
+}
+
+do_install_append () {
+    #Based on the debian/rules file, move the conf file to /etc/openjdk-11 and create a symbolic link.
+    cp ${S}/debian/accessibility.properties \
+    ${S}/debian/swing.properties \
+    ${D}${DEBIAN_JDK_SYSCONF_DIR}/
+    ln -rs ${D}${DEBIAN_JDK_SYSCONF_DIR}/accessibility.properties ${D}${DEBIAN_JDK_BASE_DIR}/conf/accessibility.properties
+    ln -rs ${D}${DEBIAN_JDK_SYSCONF_DIR}/swing.properties ${D}${DEBIAN_JDK_BASE_DIR}/conf/swing.properties
+    cp  ${S}/debian/jvm.cfg-default ${D}${DEBIAN_JDK_SYSCONF_DIR}/jvm-arm64.cfg
+    mv ${D}${DEBIAN_JDK_BASE_DIR}/lib/jvm.cfg ${D}${DEBIAN_JDK_BASE_DIR}/lib/jvm.cfg-default
```

Comment:
mv command may cause "host-user-contaminated" QA warning, so can you change to copy then remove steps?
also, cp command  cause  "host-user-contaminated" QA warning. Please use cp with --no-preserve=ownership option.  
cp's -p option may be fine.

reference:
- https://github.com/meta-debian/meta-debian/pull/280
- https://docs.yoctoproject.org/ref-manual/tasks.html#do-install

Reply:
Fix cp command option to use --no-preserve=ownership option.
Fix to copy then remove instead of mv command.